studio/data/fdw/fdws-query.ts

```diff
@@ -1,15 +1,20 @@
 import { UseQueryOptions } from '@tanstack/react-query'
 import { ExecuteSqlData, useExecuteSqlPrefetch, useExecuteSqlQuery } from '../sql/execute-sql-query'
 
+// [Joshen] Think twice - do we really need the columns here?
+// This query can just be specifically scoped to the FDW UI
```

Comment:
Yep! I put everything in this query with the intention of scoping it down once we knew what parts we needed. Feel free to remove parts as you see fit!

Reply:
oh yeah - forgot to remove this comment haha, yeah we need the columns to support the edit fdw flow